src/main/java/org/folio/rest/migration/model/request/holdings/HoldingsJob.java

```diff
@@ -14,9 +15,12 @@
 
   private Map<String, String> references;
 
+  private Map<String, List<String>> exclusions;
+
   public HoldingsJob() {
     super();
     references = new HashMap<String, String>();
```

Comment:
Might as well update this one as well.
```suggestion
    references = new HashMap<>();
```